net/nginx/Makefile

```diff
@@ -71,6 +71,7 @@ PKG_CONFIG_DEPENDS := \
 	CONFIG_NGINX_STREAM_SSL_PREREAD_MODULE \
 	CONFIG_NGINX_RTMP_MODULE \
 	CONFIG_NGINX_TS_MODULE \
+	CONFIG_OPENSSL_ENGINE \
```

Comment:
This is what I meant.  Notice that it comes with a side effect: nginx will be rebuilt if the `OPENSSL_ENGINE` option is changed even if SSL is turned off.  Considering the fact that SSL should _almost_ always be used, I'd leave it here like this.

Reply:
IMHO not a problem.

Reply:
Typo fixed